pom.xml

```diff
@@ -49,7 +48,9 @@
         <!--Cornell Customization: backport redis-->
         <spring-data-redis.version>2.6.0</spring-data-redis.version>
         <slf4j-api.version>1.7.32</slf4j-api.version>
+        <squiggly-filter-jackson.version>1.3.16</squiggly-filter-jackson.version>
         <spring.version>5.3.19</spring.version>
+        <spring-security.version>5.6.3</spring-security.version>
```

Comment:
Once you've rebased this feature branch to resolve the merge conflicts, you may want to change this dependency's version to 5.6.4 or 5.6.5, so that its transitive dependencies line up with the recent KFSPTS-25000 changes.

Reply:
yes